test/runtests.jl

```diff
@@ -98,4 +98,9 @@ end
 
 @testset "Utilities" begin
     @test sprint(show, InvertedIndices.InvertedIndexIterator([2,4], 1:5)) == "[1, 3, 5]"
+    A = [1]
+    ex = try A[Not(2,3)] catch ex; ex end
+    buf = IOBuffer()
+    Base.showerror(buf, ex)
```

Comment:
I think you can just make this `sprint(showerror, ex)`

Reply:
Thanks!